doc/api.yml

```diff
@@ -1072,6 +1291,99 @@ components:
       properties:
         main_teacher:
           $ref: '#/components/schemas/Teacher'
+    Order:
+      type: string
+      enum:
+        - ASC
+        - DESC
+
+    ExamInfo:
+      type: object
+      properties:
+        title:
+          type: string
+        date:
+          type: string
+          format: date-time
+    Exam:
+      allOf:
+        - $ref: '#/components/schemas/ExamInfo'
+      properties:
+        id:
+          type: string
+        coefficient:
+          type: integer
+
+    ExamDetail:
+      type: object
+      properties:
+        id:
+          type: string
+        infos:
+          $ref: '#/components/schemas/ExamInfo'
+        participants:
+          type: array
+          items:
+            $ref: '#/components/schemas/Student'
+        student_grades:
+          type: array
+          items:
+            $ref: '#/components/schemas/StudentGrade'
+    Grade:
+      type: object
+      properties:
+        id:
+          type: string
+        point:
+          type: integer
+          example: 12
+        created_at:
+          type: string
+          format: date-time
+    StudentGrade:
+      type: object
+      properties:
+        id:
+          type: string
+        student_id:
```

Comment:
No.

If you want to use some attributes from Student, create a sub-resource that will be used inside Student.